ext/greenwich/scss/_tweaks.scss

```diff
@@ -0,0 +1,16 @@
+/* Put an outer border around the checkbox&label - similar to the borders on form-controls */
+.checkbox-inline {
+  // height: $input-height-base; // Make inputs at least the height of their button counterpart (base line-height + padding + border)
+  padding: $padding-base-vertical $padding-base-horizontal $padding-base-vertical (20+$padding-base-horizontal);
+  // font-size: $font-size-base;
+  // line-height: $line-height-base;
```

Comment:
These are copied from the `.form-control` rule. I commented out a few  blurbs because I was checking to see which ones were really necessary. Could flip a coin on whether it's better to reproduce or remove these bits.